src/main/java/adris/altoclef/tasks/speedrun/BeatMinecraft2Task.java

```diff
@@ -560,17 +578,27 @@ private Task getEyesOfEnderTask(AltoClef mod, int targetEyes) {
                         return new EquipArmorTask(COLLECT_EYE_ARMOR);
                     }
                 }
+                if(_lootTask != null && !_lootTask.isFinished(mod)) {
```

Comment:
Usually we do `task.isActive() && !_task.isFinished(mod)`

Reply:
Oooh yeah for some weird reason `_lootTask` always said it was active and I wasn't really sure how to solve that so